src/runtime/entity.ts

```diff
@@ -22,7 +22,12 @@ import {Storable} from './storable.js';
 
 export type EntityRawData = {};
 
-export type SerializedEntity = {id: string, expirationTimestamp?: string, rawData: EntityRawData};
+export type SerializedEntity = {
+  id: string,
+  creationTimestamp: string,
```

Comment:
general question, why do we use string? wouldn't Date be more appropriate?

Reply:
i think this is because i was too traumatized by the platform compatibility of dates in kt, when i was adding expiration. :(
shall i convert it? long or Date?

Reply:
I think Date would be best. It is also fine to just leave a todo for now.

Reply:
added a TODO (whatever we do, i'd prefer to do it as a follow up PR). 
thanks!